packages/codebytes/src/__tests__/mocks.ts

```diff
@@ -0,0 +1,8 @@
+jest.mock('react-resize-observer');
+jest.mock('../libs/eventTracking');
+jest.mock('../helpers', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('../helpers'),
+  trackClick: jest.fn(),
+}));
```

Comment:
Making a shared files for the mocks that are used both in the codebytes-test and editor-test